addons/website_gengo/static/src/css/website_gengo.css

```diff
@@ -1,3 +1,3 @@
-.o_translatable_inprogress {
-  background: #b7e4ff;
-}
\ No newline at end of file
+html[lang] > body.editor_enable [data-oe-translation-state="inprogress"] {
```

Comment:
Its parent already uses ```!important```, to override the effect of its parent, we also required to use ```!important``` here.
```
html[lang] > body.editor_enable [data-oe-translation-state] {
  background: rgba(255, 255, 90, 0.5) !important;
}
```
https://github.com/odoo/odoo/blob/master/addons/web_editor/static/src/css/editor.css#L835